Add a metaTitle and a metaDescription to every doc pages (#233)
strapi/documentation: docs/user-docs/latest/content-types-builder/introduction-to-content-types-builder.md

---
title: Content-Types Builder - Strapi User Guide

> description: From the Content-Types Builder, administrators can create and manage content-types: collection types and single types but also components.

Reviewer: ```suggestion
description: Introduction to the Content-Types Builder where administrators can create and manage collection types, single types and components.
```
